src/odemis/acq/drift/__init__.py

```diff
@@ -216,7 +216,7 @@ def estimateCorrectionPeriod(period, t, repetitions):
         :param t: (float) Time spend for acquiring one data at the fastest dimension.
         :param repetitions: (tuple of 2 ints) Number of iterations for the two fastest axes in the
                                               entire drift-corrected acquisition. First value is the
-                                              fastest dimension scanned. Usually it is the
+                                              slowest dimension scanned. Usually it is the
                                               number of px (ebeam) positions. Sometimes fastest dimension
                                               can be e.g. number of images to integrate.
```

Comment:
I'm not sure if I understand the rest of the comment, should that now be the following?
```suggestion
                                              slowest dimension scanned. Usually the fastest dimension is the
                                              number of px (ebeam) positions, sometimes it
                                              can be e.g. number of images to integrate.
```

Reply:
Good catch, the whole text is actually about the "first dimension", which is the slowest, but I had updated only half of it. I've now updated it further.